### PR TITLE
new docs

### PR DESCRIPTION
**Pull request by MoElkhidir**

### CODE REVIEW

**Review by au-re — CHANGES_REQUESTED** [review]
let's sync on the concept document first

File: websites/docs/pages/index.md [hunk 1]
```diff
@@ -1,51 +1,11 @@
...
-## Use Cases
-
-LLMs excel in tasks such as personalized customer service, efficient data analysis and synthesis, automating complex workflows, and providing intuitive interfaces for human-machine interaction. Below are some of the things you can build with Prompt Studio:
-
-- Document Processing (unstructured text to structured data)
-- Retrieval + Q&A
-- Quick prototyping of AI workflows
-- Conversational Systems
+::: tip Did you know?
+The Prompt Studio Docs content and navigation is fully generated by a Prompt Studio recipe!
```
Comment (SaraFatih) [inline websites/docs/pages/index.md:10]: Shall we link to the page about "recipe" from here?

File: websites/docs/pages/key-concepts/datapoint.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Datapoint
```
Comment (SaraFatih) [inline websites/docs/pages/key-concepts/datapoint.md:1]: Can we say that a Datapoint is a record of a Recipe's execution at a given moment in history [and we refer to a place called history]. We'll need to add a place that explains what history is. And history can be referring to datapoints as well.

File: websites/docs/pages/key-concepts/recipe/content.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Content
+Content within a Recipe refers to static text elements that are not processed by the AI but are essential for the workflow. This for example could include email templates or contextual information that supports the AI in generating more accurate outputs.
```
Comment (SaraFatih) [inline websites/docs/pages/key-concepts/recipe/content.md:2]: Content within a Recipe refers to text or context that supports the AI in generating more accurate outputs. You can upload a file as Content.

File: websites/docs/pages/key-concepts/recipe/deployment.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Deployment
+Deployment is the final step in the Recipe lifecycle. Once tested and confirmed, a Recipe can be published for use by others or integrated into external applications via the Prompt Studio API or SDK.
```
Comment (SaraFatih) [inline websites/docs/pages/key-concepts/recipe/deployment.md:2]: Let's refer to the API and SDK pages from here

File: websites/docs/pages/key-concepts/recipe/index.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+A Recipe in Prompt Studio is the blueprint for an AI-driven process. It's a structured set of instructions and content that, when executed, performs a specific task or series of tasks to deliver business value. 
+
+Below are the components that make up a Recipe:
+
+* [Instruction](./instruction.md)
+* [Content](./content.md)
+* [AI-Setting](AI-Setting.md)
+* [Preview](./preview.md)
```
Comment (SaraFatih) [inline websites/docs/pages/key-concepts/recipe/index.md:8]: Are Preview and deployment really a part of a recipe? I feel like they're a separate entity.

File: websites/docs/pages/key-concepts/scenarios.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+# Scenarios
+Scenarios enable batch execution of a Recipe, allowing users to process large datasets or historical data efficiently. This feature is crucial for testing Recipes under various conditions and inputs.
```
Comment (SaraFatih) [inline websites/docs/pages/key-concepts/scenarios.md:2]: What do you mean here with historical data?

File: websites/docs/pages/sdk-integration/index.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+run();
+```
+
+Replace `'your-deployment-id'` with the actual ID of your deployment and provide the necessary input data.
+
+
+## Support
+
+If you encounter any issues or have questions regarding the SDK integration, please reach out to our support team or visit the FAQ section.
```
Comment (SaraFatih) [inline websites/docs/pages/sdk-integration/index.md:59]: let's add our support team's email here: support@prompt.studio

**Review by SaraFatih — CHANGES_REQUESTED** [review]

File: websites/docs/pages/index.md (hunk 1 above)
Comment (MoElkhidir) [reply websites/docs/pages/index.md:10]: I don't think that's even technically allowed yet unfortunately, but the moment we allow global view-only access to some recipes, we can update this with a link!

File: websites/docs/pages/key-concepts/recipe/content.md (hunk 3 above)
Comment (MoElkhidir) [reply websites/docs/pages/key-concepts/recipe/content.md:2]: I think the clarification that's not processed by the AI is important for the user to understand. Good point about the file as well!

File: websites/docs/pages/key-concepts/recipe/index.md (hunk 5 above)
Comment (MoElkhidir) [reply websites/docs/pages/key-concepts/recipe/index.md:8]: they're directly linked to a recipe. You can preview a recipe and you can deploy it, they are tightly coupled to it and you can't create them outside the recipe page (at least for now).

File: websites/docs/pages/key-concepts/scenarios.md (hunk 6 above)
Comment (MoElkhidir) [reply websites/docs/pages/key-concepts/scenarios.md:2]: for example the drp emails, it's data that they already have and they want to upload it and classify it. This is in contrast to plugins for example where the data is new.

**Pull request closed without merging**